yarn-project/simulator/src/public/index.test.ts

```diff
@@ -618,4 +632,110 @@ describe('ACIR public execution simulator', () => {
       });
     });
   });
+
+  describe('Global variables in public context', () => {
```

Comment:
Should probably add the same in private to be thorough.

Reply:
Will add the more thorough header tests in a separate PR.